docs/source/get-started.md

```diff
@@ -0,0 +1,154 @@
+    id
+    repository {
+      name
+    }
+    postedBy {
+      login
+    }
+  }
+}
+```
+
+> Note: For the sample project you can find the `.graphql` files here => `apollo-sample/src/main/graphql/com/apollographql/apollo/sample`.
+
+<h2 id="code-generation">Code Generation</h2>
+
+To have Apollo generate the appropriate Java classes for you, you will need to compile your code. Based on your downloaded `schema.json` file and the contents of your `.graphql` files you will have generated Java classes in the following directory `build/generated/source/apollo`. There will be one Java class for each of your queries with nested classes for reading the network response.
+
+![Generated Class Structure](generated_feed_query_class_structure.png)
+
+> Note: These are Apollo generated files. Therefore they **should not** be modified.
+
+<h2 id="consuming-code">Consuming Code</h2>
+
+You can use the generated classes to make requests to your GraphQL API. `ApolloClient` can take as input any *Query* or *Mutation* that you have generated using Apollo.
+
+First, you would create an instance of your desired query or mutation by using the associated `builder()` method. This Builder class was generated for you as part of the Apollo code gen process.
+
+```java
+FeedQuery feedQuery = FeedQuery.builder()
+                               .limit(10)
+                               .type(FeedType.HOT)
+                               .build()
+```
+
+Then you can pass that query into the `query()` method available on the `ApolloClient` instance and enqueue your request. Once the request is completed you will have access to the data inside of the `onResponse()` method.
+
+```java
+apolloClient.query(feedQuery).enqueue(new ApolloCall.Callback<FeedQuery.Data>() {
+
+  @Override public void onResponse(@NotNull Response<FeedQuery.Data> dataResponse) {
+
+    final StringBuffer buffer = new StringBuffer();
+    for (FeedQuery.Data.Feed feed : dataResponse.data().feed()) {
+      buffer.append("name:" + feed.repository().fragments().repositoryFragment().name());
+      buffer.append(" owner: " + feed.repository().fragments().repositoryFragment().owner().login());
+      buffer.append(" postedBy: " + feed.postedBy().login());
+      buffer.append("\n~~~~~~~~~~~");
+      buffer.append("\n\n");
+    }
+
+    MainActivity.this.runOnUiThread(new Runnable() {
+      @Override public void run() {
+        TextView txtResponse = (TextView) findViewById(R.id.txtResponse);
+        txtResponse.setText(buffer.toString());
+      }
+    });
+      
+  }
+
+  @Override public void onFailure(@NotNull Throwable t) {
+    Log.e(TAG, t.getMessage(), t);
+  }
+});       
+```
+
+In this example, we loop over the results and append them to a `StringBuffer`. Finally, we display the string inside of a `TextView` on the main thread.
+
+> Note: `onResponse()` returns on a background thread. If you want to make UI updates make sure they are done on the Main Thread.
```

Comment:
There's an android support package that lets you deliver on main thread if you want to mention that

Reply:
I'm thinking we should create a `Recipes` page which includes info about how to use the `android-support` package.